extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/channels.xml

```diff
@@ -332,11 +334,25 @@
 		<state readOnly="true" pattern="%d d" />
 	</channel-type>
 
+	<channel-type id="ageDegree">
+		<item-type>Number</item-type>
+		<label>Moon age in degree</label>
+		<description>The age of the moon in degree</description>
+		<state readOnly="true" pattern="%.2f °" />
+	</channel-type>
+
+	<channel-type id="agePercent">
+		<item-type>Number</item-type>
+		<label>Moon age in percent</label>
```

Comment:
Note that labels should be as short as possible (best 1-2 words only).
As the unit is shown in the state anyhow, couldn't this simply be "Moon age"?